src/main/java/gregtech/common/blocks/FrameItemBlock.java

```diff
@@ -33,4 +37,12 @@ public String getItemStackDisplayName(@Nonnull ItemStack stack) {
         Material material = getBlockState(stack).getValue(frameBlock.variantProperty);
         return OrePrefix.frameGt.getLocalNameForItem(material);
     }
+
+    @Override
+    public void addInformation(ItemStack stack, @Nullable World worldIn, List<String> tooltip, ITooltipFlag flagIn) {
+        super.addInformation(stack, worldIn, tooltip, flagIn);
+        if (flagIn.isAdvanced()) {
+            tooltip.add("MetaItem Id: block" + frameBlock.getGtMaterial(stack.getMetadata()).toCamelCaseString());
```

Comment:
frame?

Reply:
yeah, i noticed that after looking over it. Will change it later